packages/bullet/src/types.ts

```diff
@@ -0,0 +1,189 @@
+import * as React from 'react'
+import { Box, Dimensions, Theme, Colors, ModernMotionProps } from '@nivo/core'
+import { ScaleLinear } from 'd3-scale'
+import { AnimatedValue } from 'react-spring'
+
+export type DatumId = string
+export type DatumValue = number
+
+export type WithDatumId<R> = R & { id: DatumId }
+
+type Point = {
+    x: number
+    y: number
+}
+
+export interface DefaultRawDatum {
```

Comment:
I renamed it to Datum. What generics are you talking about here? I have a couple for event handlers, because one accepts Range/Measure data and the other accepts Marker data , which shapes differ.